### DISCUSSION TITLE
sqlite3 Storage class using sqlite3 json entries

### DISCUSSION THREAD

**Discussion by MatthiasLesch** [msiemens/tinydb#578] [General]
Naive question: are there plans/ideas to add a sqlite3 storage class. The actual document data could be stored still as json as sqlite3 supports json table fields.

**Comment by msiemens** [comment]
There are no specific plans to add further storage engines to TinyDB core, but I'm sure there is room for a community extension! I'd definitely add it to the [extensions page](https://tinydb.readthedocs.io/en/latest/extensions.html) of the documentation 🙂
